RATapi/models.py

```diff
@@ -62,6 +62,18 @@ class Background(RATModel):
     value_4: str = ""
     value_5: str = ""
 
+    @model_validator(mode="after")
+    def validate_unimplemented_backgrounds(self):
+        """Raise an error if currently unsupported Data or Function backgrounds are used."""
+        # FIXME: once data/function backgrounds have been implemented,
+        # please remember to remove the @pytest.mark.skip decorators used to skip the tests:
+        # - tests/test_project.py::test_check_allowed_background_resolution_values_data
+        # - tests/test_project.py::test_check_allowed_background_resolution_values_on_data_list
+        if self.type == TypeOptions.Data:
+            raise NotImplementedError("Data backgrounds are not yet supported.")
+        elif self.type == TypeOptions.Function:
+            raise NotImplementedError("Function backgrounds are not yet supported.")
```

Comment:
This looks good and does the job, but why not use a field validator since it's only the `type` that matters here?

Reply:
good spot - was originally using a model validator because it was also checking whether `value_1` was filled